src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -974,6 +974,12 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         self._autoButtons = _AutoScaleButton(self)
         self._autoButtons.autoRangeChanged.connect(self._autoRangeButtonsUpdated)
 
+        self._saturation = qt.QSlider(qt.Qt.Horizontal, self)
+        self._saturation.setTickPosition(qt.QSlider.TicksBelow)
+        self._saturation.setRange(0, 100)
+        self._saturation.setValue(2)  # 2 <=> 1-99 percentile mode
+        self._saturation.valueChanged.connect(self._saturationChanged)
```

Comment:
Probably good to disable tracking here, or it's going to lag

https://doc.qt.io/qt-6/qabstractslider.html#tracking-prop

Reply:
Agree but then to come back on the previous discussion the slider is also loosing of his interest...